### ISSUE TITLE
Accessibility considerations for year sliders

### ISSUE THREAD

**Issue by CAYdenberg** [owid/owid-grapher#3137]
## Description

A number of charts feature year sliders that allow the user to pan back and forth in time. While not formally broken, the behaviour of these elements is mysterious (actions have unexpected consequences), and are also inaccessible by keyboard control.

## Expected behaviour

There should be clear and well labelled ways to convert between range and single-year mode, and controls should be accessible to keyboard users.

## Steps to reproduce

Go to https://ourworldindata.org/grapher/co2-emissions-vs-gdp and mess with the slider.

The only way to convert from single-year to range mode is to click one of the labels. The only way to convert back to single-year is to slide the handles on top of each other. These consequences are unexpected. Proper HTML form controls would be much simpler, easier to understand, and usable by keyboard.

## Environment

The same experience is present on all devices.

**Comment by danyx23** [comment]
There is a half-finished branch by @sophiamersmann that we'd like to finish at some point that adds an explicit tab to switch between single-year and multi-year view

**Comment by danyx23** [comment]
Switching between single year chart view (bar chart) and multiple year view (line chart) will likely be added in the next few months (related to #3231 and #1162). 

The accessibility concern is a very good point and we should implement that part of the issue soon as well.
